.circleci/resources/pytest_build_config.ini

```diff
@@ -0,0 +1,6 @@
+#pytest.ini
+[tool:pytest]
+minversion = 1.0
+addopts = -ra -q
+testpaths =
+    autodh/tests
```

Comment:
can you move the tests to the root directory of the repo. That's the pattern we follow everywhere else